Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1150,27 +1151,38 @@ theorem bddAbove_range {ι : Type u} (f : ι → Ordinal.{max u v}) : BddAbove (
     exact le_of_lt (Cardinal.lt_ord.2 ((lt_succ _).trans_le
       (le_ciSup (Cardinal.bddAbove_range.{_, v} _) _)))⟩
 
-/-- `le_ciSup` whenever the outputs live in a higher universe than the inputs. -/
-protected theorem le_iSup {ι : Type u} (f : ι → Ordinal.{max u v}) : ∀ i, f i ≤ iSup f :=
-  le_ciSup (bddAbove_range f)
+theorem bddAbove_of_small (s : Set Ordinal.{u}) [h : Small.{u} s] : BddAbove s := by
```

Comment:
I know you are only moving this lemma, but I would suggest
```suggestion
theorem bddAbove_of_small {s : Set Ordinal.{u}} [h : Small.{u} s] : BddAbove s := by
```
since most often you don't have to specify what `s` is

Reply:
I don't think I agree with that, I expect the pattern `obtain (a, ha) := bddAbove_of_small (complicatedSet)` to be quite useful. 

This also matches `Cardinal.bddAbove_of_small` which also uses an explicit arg.

Reply:
Okay, the pattern you're referring to happens exactly once so far: `obtain ⟨a, ha⟩ := bddAbove_of_small (enumOrd s '' Iio o)` in `SetTheory.Ordinal.Enum`, while `bddAbove_of_small _` appears five times. I guess we can wait a bit longer before making the argument implicit